### DISCUSSION TITLE
Add Panoramax project on links screen

### DISCUSSION THREAD

**Discussion by Jean-BaptisteC** [streetcomplete/StreetComplete#5577] [Ideas]
A new project similar to Mapillary (based on Geovisio) has been created in France.
Panoramax is built around different instances, but a catalog with all photos from all instances is available at this URL https://api.panoramax.xyz
For now, the project is focused in France, but some contributors started to publish photos in other countries.

WDYT to add this project on links screen? (we have already KartaView and Mapillary)

**Comment by westnordost** [comment]
I would love to, and then I would probably also remove Mapillary and KartaView, but it only starts to be useful for users once there is an official go-to place to upload photos to. Otherwise, it is just interesting for devs and for maybe local OSM chapters.

**Reply by westnordost** [reply]
I mean, a global one, not just France, and not just (for the moment) tolerated by OSM France.

**Comment by JLZIMMERMANN** [comment]
> I would love to, and then I would probably also remove Mapillary and KartaView, but it only starts to be useful for users once there is an official go-to place to upload photos to. Otherwise, it is just interesting for devs and for maybe local OSM chapters.

If only it could be possible to use it in France.

**Reply by westnordost** [reply]
Why is it not possible to use it in France?

**Reply by JLZIMMERMANN** [reply]
My idea is to let user to choose Panoramax to upload pictures.

**Comment by Strubbl** [comment]
~~Creating an instance for StreetComplete only is not an option? Analog to the mapcomplete instance? Is storage for the images the limiting factor? Or is it any legal issue? ATM, the images are deleted once the note is resolved because of?
Or maybe we could use the mapcomplete instance with StreetComplete if we ask the mapcomplete team?~~

edit: Sorry, i think my comment is off-topic. I should create a new discussion for this. Only understood now that it is only about linking. → done at https://github.com/streetcomplete/StreetComplete/discussions/6467
